### PR TITLE
Group event calendar

### PR DESCRIPTION
**Pull request by laura-voss**

### CODE REVIEW

File: client/src/components/groupCalendar.js [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+        date: new Date(value),
+        name,
+        user_id: userArray,
+      });
+    } finally {
+      if (response) {
+        alert(`${name} is booked in`);
+        navigate('/home');
+      } else {
+        alert('try again... muhahahah');
```
Comment (AKCDNG) [inline client/src/components/groupCalendar.js:148]: Alert shall be changed to a dynamic error on page

**Review by AKCDNG — APPROVED** [review]
Couple changes need to be made, but nothing too serious. Made notes. Can merge and make changes later. Also many comments that we will need to go through later on during the week to see if they are necessary or not.

**Comment by amfibiya17** [comment]
Looks good for me

**Review by Rmorbey — APPROVED** [review]
It looks all good to me.

**Pull request merged**